src/som/compiler/MixinDefinition.java

```diff
@@ -73,6 +72,7 @@
  * at runtime, which then also has the super class and mixins resolved to be
  * used to instantiate {@link SClass} objects.
  */
+@ExportLibrary(InteropLibrary.class)
```

Comment:
Why add `@ExportLibrary(InteropLibrary.class)` to this class if you don't implement any of the interop messages (with `@ExportMessage`)?

Reply:
Work in progress

Reply:
Ok, I was just wondering because this is not a draft pull request. 😉

Reply:
This is not going to be addressed in this PR. There's no need for more interop support.